games-emulation/gambatte-jg/gambatte-jg-0.6.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+MY_PN=${PN%-*}
+MY_P=${MY_PN}-${PV}
+DESCRIPTION="Jolly Good Fork of Gambatte"
+HOMEPAGE="https://gitlab.com/jgemu/gambatte"
+if [[ "${PV}" == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/jgemu/${MY_PN}.git"
+else
+	SRC_URI="https://gitlab.com/jgemu/${MY_PN}/-/archive/${PV}/${MY_P}.tar.bz2"
+	S="${WORKDIR}/${MY_P}"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+fi
+
+LICENSE="
+	GPL-2
+	examples? ( 0BSD )
+	jgmodule? ( BSD )
+"
+SLOT="1"
+IUSE="examples +jgmodule shared"
+REQUIRED_USE="|| ( examples jgmodule shared )"
+
+DEPEND="
+	examples? (
+		media-libs/libsdl2[sound,video]
+		media-libs/speexdsp
+	)
+	jgmodule? (
+		media-libs/jg:1=
+		media-libs/soxr
+	)
+"
+RDEPEND="
+	${DEPEND}
+	jgmodule? ( games-emulation/jgrf )
+"
+BDEPEND="
+	virtual/pkgconfig
+"
+
+pkg_setup() {
```

Comment:
This is normally done in src_configure, but if there's no ./configure here, you can as well do it in src_compile

Reply:
Using `pkg_setup` to set the make arguments was originally copied from `dev-vcs/git`, but the git ebuild seems to have changed to use `src_configure` since I did so.

So I changed it to use `src_configure`  here too, I hope that is acceptable?